src/main/java/frc/robot/subsystems/led/LedSubsystem.java

```diff
@@ -0,0 +1,86 @@
+// Copyright (c) FIRST and other WPILib contributors.
+// Open Source Software; you can modify and/or share it under the terms of
+// the WPILib BSD license file in the root directory of this project.
+
+package frc.robot.subsystems.led;
+
+import java.util.Map;
+
+import edu.wpi.first.networktables.GenericEntry;
+import edu.wpi.first.wpilibj.AddressableLED;
+import edu.wpi.first.wpilibj.AddressableLEDBuffer;
+import edu.wpi.first.wpilibj.shuffleboard.BuiltInWidgets;
+import edu.wpi.first.wpilibj.shuffleboard.Shuffleboard;
+import edu.wpi.first.wpilibj.shuffleboard.ShuffleboardTab;
+import edu.wpi.first.wpilibj.shuffleboard.SimpleWidget;
+import edu.wpi.first.wpilibj.util.Color;
+import edu.wpi.first.wpilibj2.command.SubsystemBase;
+import frc.robot.Robot;
+public class LedSubsystem extends SubsystemBase {
+  private final AddressableLEDBuffer buffer;
+  private final AddressableLED leds;
+  private ShuffleboardTab ledTab = Shuffleboard.getTab("Led");
+  private GenericEntry ledState = ledTab.add("OnOff", false).getEntry();
+  private SimpleWidget ledGamePiece = ledTab.add("Color", true).withWidget(BuiltInWidgets.kBooleanBox).withProperties(Map.of("colorWhenFalse", "Purple", "colorWhenTrue", "Yellow"));
+  private double ledBrightnessSlider = ledTab.add("Brightness", 0.2).withWidget(BuiltInWidgets.kNumberSlider).getEntry().getDouble(0.2);
+
+  public static enum LedMode {
+    CONE, CUBE;
+  }
+  private LedMode ledMode = LedMode.CONE;
+  /** Configure AddressableLED */
+  public LedSubsystem() {
```

Comment:
Rename to LedStripSubsystem, makes differentiating between the subsystems easier

Reply:
resolved